infra.tf

```diff
@@ -104,6 +104,16 @@ resource "aws_launch_template" "rancher_master" {
     DoNotDelete = "true"
     Owner       = "EIO_Demo"
   }
+
+  tag_specifications {
+    resource_type = "instance"
+
+    tags = {
+      Name        = "${local.name}-master"
+      DoNotDelete = "true"
+      Owner       = "EIO_Demo"
```

Comment:
Might want to think about making these tags configurable at some point.

Reply:
Yeah for sure. I'll open an issue to track because I have the attention span of a squirrel